java/src/main/native/CMakeLists.txt

```diff
@@ -129,6 +133,7 @@ set(CUDF_INCLUDE "${PROJECT_SOURCE_DIR}/../../../../cpp/include"
 
 find_library(CUDF_LIBRARY "cudf"
     HINTS "$ENV{CUDF_ROOT}/lib"
+          "$ENV{CUDF_ROOT}"
```

Comment:
`CUDF_ROOT` is supposed to be set to the install prefix of cudf, so I don't think we want to be looking both in `CUDF_ROOT` _and_ `CUDF_ROOT/lib`.